### PR TITLE
chore: omit comparison to bool constant

### PR DESCRIPTION
**Pull request by testwill**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
should omit comparison to bool constant, can be simplified to found
#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kubelet/kubelet_server_journal.go [hunk 1]
```diff
@@ -354,7 +354,7 @@ func copyFileLogs(ctx context.Context, w io.Writer, services []string) {
 // in that order stopping on first success.
 func heuristicsCopyFileLogs(ctx context.Context, w io.Writer, service string) {
 	logFileNames := [3]string{
-		fmt.Sprintf("%s", service),
```
Comment (bart0sh) [inline pkg/kubelet/kubelet_server_journal.go:357]: How this is related to the PR?

File: test/e2e/storage/vsphere/vsphere_volume_diskformat.go [hunk 2]
```diff
@@ -194,15 +194,15 @@ func verifyDiskFormat(ctx context.Context, client clientset.Interface, nodeName
 	}
 	isDiskFormatCorrect := false
 	if diskFormat == "eagerzeroedthick" {
-		if eagerlyScrub == true && thinProvisioned == false {
+		if eagerlyScrub && !thinProvisioned {
 			isDiskFormatCorrect = true
 		}
```
Comment (bart0sh) [inline test/e2e/storage/vsphere/vsphere_volume_diskformat.go:199]:
```suggestion
		isDiskFormatCorrect = eagerlyScrub && !thinProvisioned
```

File: test/e2e/storage/vsphere/vsphere_volume_diskformat.go [hunk 3]
```diff
@@ -194,15 +194,15 @@ func verifyDiskFormat(ctx context.Context, client clientset.Interface, nodeName
...
 	} else if diskFormat == "zeroedthick" {
-		if eagerlyScrub == false && thinProvisioned == false {
+		if !eagerlyScrub && !thinProvisioned {
 			isDiskFormatCorrect = true
 		}
 	} else if diskFormat == "thin" {
-		if eagerlyScrub == false && thinProvisioned == true {
+		if !eagerlyScrub && thinProvisioned {
 			isDiskFormatCorrect = true
 		}
```
Comment (bart0sh) [inline test/e2e/storage/vsphere/vsphere_volume_diskformat.go:207]:
```suggestion
                isDiskFormatCorrect = !eagerlyScrub && thinProvisioned
```

File: test/e2e/storage/vsphere/vsphere_volume_diskformat.go [hunk 4]
```diff
@@ -194,15 +194,15 @@ func verifyDiskFormat(ctx context.Context, client clientset.Interface, nodeName
 	}
 	isDiskFormatCorrect := false
 	if diskFormat == "eagerzeroedthick" {
```
Comment (bart0sh) [inline test/e2e/storage/vsphere/vsphere_volume_diskformat.go:196]: Would it make sense to convert this `if` into a `switch` ?

**Review by bart0sh — CHANGES_REQUESTED** [review]

**Review by marosset — APPROVED** [review]

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

trivial edit, ok for node

**Review by mrunalp — APPROVED** [review]

**Comment by SergeyKanzhelev** [comment]
@msau42 can you approve for storage?

**Pull request merged**